src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -22,4 +23,4 @@
     <tr><td><a href="${pageContext.request.contextPath}/cars/drivers/add">Add Driver to Car</a></td></tr>
```

Comment:
In my opinion better to add there href to see all cars to authorised driver (GetMyCurrentCarsController)